integration-tests/java/build.gradle

```diff
@@ -49,6 +49,7 @@ task integrationTest(type: Test) {
 
     testClassesDirs = sourceSets.integrationTest.output.classesDirs
     classpath = sourceSets.integrationTest.runtimeClasspath
+    outputs.upToDateWhen { false }
```

Comment:
Another option is:

./gradlew cleanIntegrationTest integrationTest

But perhaps you can take a quick read of this advice to see if there is a recommended way:
https://blog.gradle.org/stop-rerunning-tests

Reply:
The blog post make sense. We should design how often it should be run. Since this is integration tests, it is supposed to be run at least every day. 
Reflecting back to my original intention, I added it to rerun it during my development, which is not appropriate way to use the function. I should have used  cleanIntegrationTask or --rerun-nstasks instead. 
I removed the code.